enterprise/server/backends/distributed/distributed.go

```diff
@@ -19,26 +20,46 @@ import (
 )
 
 type Cache struct {
-	local     interfaces.Cache
-	myAddr    string
-	groupName string
-	prefix    string
+	local             interfaces.Cache
+	myAddr            string
```

Comment:
I added detailed comments on the public New method.

(since we're not really commenting everything -- I think commenting public methods is probably a decent place to start)